src/Messaging/Dispatchers/PrintConsignmentNoteDispatcher.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace Linn.Stores.Messaging.Dispatchers
+{
+    using System.Text;
+
+    using Linn.Common.Messaging.RabbitMQ;
+    using Linn.Stores.Domain.LinnApps.Dispatchers;
+    using Linn.Stores.Resources.MessageDispatch;
+
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Serialization;
+
+    public class PrintConsignmentNoteDispatcher : IPrintConsignmentNoteDispatcher
+    {
+        private const string ContentType = "application/json";
+
+        private readonly string routingKey = "orawin.consignment-note.print";
+
```

Comment:
any readon why one is a const and one is a readonly?

Reply:
not really